### PR TITLE
Theme Styles Update

### PR DESCRIPTION
**Pull request by rahilllshah**
## Overview

Update to styles file and themes file to streamline colours for the app.

### CODE REVIEW

**Comment by benjaminkostiuk** [comment]
@rahilllshah Did you run this? Cause it won't even compile for me. I think you're missing imports among other things. 
For linting you can run `npm run lint:fix` to fix the linting and `npm run format` to properly format, or add prettier to format on save if you're using vscode.

**Review by benjaminkostiuk — COMMENTED** [review]
Missing at the top of theme
```
import * as STYLES from "./styles";
```
Then everywhere you can do something like `STYLES.WHITE_COLOR_LIGHT`.

But you need to make sure you can compile to actually test your changes.

**Comment by benjaminkostiuk** [comment]
@rahilllshah Can you also check for values in `antDesignOverride` that we might need to change from the list in https://github.com/ant-design/ant-design-mobile-rn/blob/master/components/style/themes/default.tsx. There's probably highlights of buttons and hover colors Dhruv and I missed when we quickly overrode some of the values.

File: src/constants/theme.tsx [hunk 1]
```diff
@@ -77,60 +97,75 @@ export const themes = {
...
         fontSize: 14,
       },
+      placeholderText: {
+        color: STYLES.LIGHT_COLOR_DARK,
+      },
+      buttonColor: {
+        color: STYLES.PRIMARY_COLOR_DARK,
+        transition: 0.25,
+      },
+      buttonHover: {
```
Comment (dhruv423) [inline src/constants/theme.tsx:120]: You can remove this since it is no longer needed

**Review by dhruv423 — APPROVED** [review]
LGTM!

**Pull request merged**